src/main/resources/com/google/api/codegen/java/http_unit_test.snip

```diff
@@ -0,0 +1,135 @@
+@extends "java/common.snip"
+@extends "java/initcode.snip"
+
+@snippet generate(xapiTest)
```

Comment:
I would like to use the convention of an `x` prefix

Reply:
I assume this means you don't want the `x` prefix, so it's been removed.